src/python/pants/option/optionable.py

```diff
@@ -85,7 +85,7 @@ def optionable_cls(cls):
 
     @classmethod
     def is_valid_scope_name_component(cls, s: str) -> bool:
-        return cls._scope_name_component_re.match(s) is not None
+        return s == "" or cls._scope_name_component_re.match(s) is not None
```

Comment:
Best practice is to use `from pants.option.scope import GLOBAL_SCOPE` so that this isn't a magic value.

Reply:
I don't think that's appropriate here, specifically. This validation function is just "is this string a *valid* scope name", not "is this string an *existing* scope name". It exists so we can yell at subsystem authors if they give a badly-formed name.